src/browser/ui/dom/HTMLDOMPropertyConfig.js

```diff
@@ -179,7 +179,9 @@ var HTMLDOMPropertyConfig = {
     itemID: MUST_USE_ATTRIBUTE,
     itemRef: MUST_USE_ATTRIBUTE,
     // property is supported for OpenGraph in meta tags.
-    property: null
+    property: null,
+    // IE-only attribute that controls focus behavior
+    unselectable: MUST_USE_ATTRIBUTE    
```

Comment:
Lint is complaining because you have trailing spaces here. Otherwise this is good to go.